public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+
+:marked
+  The issue is that we do not know how to get a collection of `child-content` elements and store them in `myContent` so that we
+  can iterate through this collection, define the child component instances, and establish the input property binding.
+  
+  Angular provides the `@ContentChildren` annotation we can use to define that we need to get a collection of children that are used
+  as content within our parent component. The framework takes care of initializing this collection during component instantiation, 
+  detects changes, and dynamically updates the collection of children:
+  
++makeExample('component-communication/ts/fragments/contentchildren-intro.ts', 'parent-comp', null, {pnk: /(@ContentChildren)/g})
+
+:marked
+  [Jump to details](#-6-parent-to-child-communication-where-children-are-transposed-in-mdash-using-contentchildren-)
+
+  ### The Hero Job Board sample
+  
+  In this chapter, we will build a sample application that help us involve our heroes in actions when we really need their help.
+  In this app, users can act on behalf of the person in need, and also on behalf of each hero. The basic workflow is the following:
+  1. The person in need invites heroes to listen to job requests.
+  1. When there's a decent job that requires superpower, the person in need announces a job request that is received by all heroes.
+  1. Heroes can opt to undertake the job.
+  1. When having received applications for the job, the person in need assigns the job to the one of the responding heroes.
+  1. The assigned hero is notified about winning the job, the others receive a message about losing the opportunity.
+  
+  Here is a screenshot of our complete application:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/app-ui-snapshot.png" alt="UI in action")
+  
+:marked
+  ### Live Example
+  
+  The application is available as a live example. It is a single Angular app that uses the router configured with separate routing paths 
+  for each scenario. To try the sample code of a specific scenario, use the tabs right beneath the __Component Communication__ title.
+  
+  [Live Example](/resources/live-examples/component-communication/ts/plnkr.html)
+  
+  Let's start, and dive into the details!
+  
+  ## #1: Parent to child communication with input property binding
+  
+  Angular allows components to declare _data-bound input properties_. These are like regular component, but the value of an 
+  input property takes its value from the expression bound to it. The framework automatically updates data-bound input properties
+  during change detection.
+  
+  This is a _one-way binding_. As the source value changes, the destination property of the component is changed, but it does not 
+  happen in the opposite direction: altering the value of the input property within the component does not affect the source.
```

Comment:
Not precisely true.  We can modify the `Hero` object itself and that modification flows through to the parent.

What you mean to say is that we can't change the property value and thus cannot replace the hero reference.